packages/redux/src/settings/selectors/accountSettings.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  getError,
+  getIsLoading,
+  getResult,
+} from '../reducer/accountSettings.js';
+import type { AccountSettingsHash, SettingsState } from '../types/index.js';
+import type { AccountSettingsQuery } from '@farfetch/blackout-client';
+import type { StoreState } from '../../types/index.js';
+
+const hashQuery = (query?: AccountSettingsQuery): AccountSettingsHash => {
```

Comment:
Since this is being reused in `fetchApplicationSettingsFactory` module, it is better to place this function in a utils module and reuse it here and in `fetchApplicationSettingsFactory`.

Reply:
Done 💯